helm/files/oracle.yaml

```
@@ -0,0 +1,21 @@
+
+    - alert: OracleUptimeUnder50%
+      expr: |
+        (
+          (cvms_oracle_vote_window - on (chain_id) group_right () cvms_oracle_miss_counter)
+            / on (chain_id) group_left ()
+          cvms_oracle_vote_window
+        ) < 0.5
+      labels:
+        severity: critical
+      annotations:
+        summary: The Validator's Oracle vote rate is too low for the {{ $labels.chain }}-{{ $labels.chain_id }} network
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)